homeassistant/components/zwave_js/discovery.py

```diff
@@ -492,6 +495,17 @@ def get_config_parameter_discovery_schema(
             type={"any"},
         ),
     ),
+    # humidifier
+    # thermostats supporting mode (and optional setpoint)
+    ZWaveDiscoverySchema(
+        platform="humidifier",
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.HUMIDITY_CONTROL_MODE},
+            property={HUMIDITY_CONTROL_MODE_PROPERTY},
+            type={"number"},
+        ),
+        allow_multi=True,
```

Comment:
The same z-wave node will likely have a humidifier and a thermostat, which is why this discovery entry is placed above the `climate` ones, and this one has `allow_multi`. Is this the correct way of doing this?

Reply:
`allow_multi` is only needed when the same ZWave Value is used to create multiple entities. In this case I don't think it is needed.